packages/lb-annotation/src/core/scheduler.ts

```diff
@@ -346,5 +346,30 @@ export class ToolScheduler implements IToolSchedulerOperation {
   public init() {
     this.toolOperationList = [];
     this.toolOperationDom = [];
+    this.eventBinding();
+  }
+
+  public destroy() {
+    this.destroyAllLayer();
+    this.eventUnBinding();
+  }
+
+  public eventBinding() {
+    // this event will be touched when the remark layer blocks the onWheel events in specific operations, otherwise it will be blocked by the onWheel events inside
+    this.container.addEventListener('wheel', this.onWheel);
+  }
+
+  public eventUnBinding() {
+    this.container.removeEventListener('wheel', this.onWheel);
+  }
+
+  public onWheel(e: MouseEvent) {
+    e.preventDefault();
+    e.stopPropagation();
+    if (this.toolOperationList.length !== 0) {
+      const lastOneIndex = this.toolOperationDom.length - 1;
+      // the last one operation is the active one
+      this.toolOperationList[lastOneIndex].onWheel(e);
```

Comment:
Is there any problem in annotatiing ? It will trigger twice.（ zoom more quickly ?）

Reply:
If there is no remark layer, this event will be blocked by 'preventDefault' in 'onWheel' events in basicToolOperation, so it wont be triggered twice